custom_components/frigate/camera.py

```diff
@@ -120,6 +124,7 @@ def __init__(
         Camera.__init__(self)
         self._url = config_entry.data[CONF_URL]
         self._attr_is_on = True
+        self._attr_device_class = DEVICE_CLASS_CAMERA
```

Comment:
Maybe add a comment here to indicate why we're seeing the device class? (As we'll immediately forget otherwise!)

Reply:
Done.